src/typings/vendors.d.ts

```diff
@@ -3,6 +3,6 @@ declare module '*.md'
 declare module 'country-telephone-data' {
   const countryTelData: countryTelData
   const allCountries: allCountriesPhoneData[]
-  const allCountryCodes: allCountryCodes
+  const allCountryCodes: allCountriesCode
```

Comment:
why not keeping `allCountryCodes`?

Reply:
Yep it's finally better to keep the library naming

Reply:
Done